starters/basic-starter/pages/[...slug].tsx

```diff
@@ -112,9 +163,81 @@ export async function getStaticProps(
   }
 
   // Fetch the node/resource from Drupal.
-  const node = await drupal.getResourceFromContext<DrupalNode>(type, context, {
-    params: params.getQueryObject(),
-  });
+  let node;
+  if (type === 'taxonomy_term--person_type') {
+    node = await drupal.getResourceCollectionFromContext<DrupalNode[]>(
```

Comment:
I think it's confusing that we use the `node` property both for a list of nodes and for a single node. I think we should change this something else and make it so that both taxonomy term and list of nodes get passed to the component.